.github/workflows/build_and_test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     timeout-minutes: 100
     runs-on: ubuntu-20.04
     steps:
-      - name: 'Checkout p4-dpdk repo'
+      - name: 'Run Git CheckOut p4-dpdk repo'
```

Comment:
Why make it so complicated? "Check out p4-dpdk repo" is simpler and clearer.

Reply:
done!